ImageTransformationsCUDA/src/warpCUDA.jl

```diff
@@ -0,0 +1,36 @@
+using LinearAlgebra
+
+using ImageTransformations: warp, warp!
+# @Tim should I import warp and warp! instead, or continue using the ImageTransformation.warp syntax for clarity
+using ImageTransformations: autorange, try_static
+
+
+function ImageTransformations.warp(img::Union{CuArray{T,N},OffsetArray{T,N,<:CuArray}}, tform, inds::Tuple = autorange(img, inv(tform))) where {T, N}
+    out = OffsetArray(CuArray{T}(undef, map(length, inds)), inds); #Can't make CuArray of OffsetArray
+    warp!(out, img, try_static(tform, img))
+end 
+
+function ImageTransformations.warp!(out, img::OffsetArray{T,N,<:CuArray}, tform) where {T,N}
+    warp!(out, img.parent, tform, img.offsets)
+end
+
+function ImageTransformations.warp!(out, img::CuArray{T,N}, tform, in_offsets = ntuple(i->0, N)) where {T,N} #why is this now unstable?!
+    img_inds = map(out->out.I, CartesianIndices(axes(out.parent))) 
+    tform_offset = offset_calc(out, in_offsets, tform)
+
+    tformindex = CuArray(tform_offset.(SVector.(img_inds))) 
```

Comment:
This calculates _all_ image indexes in the single thread CPU and converts them to GPU. I believe this is unnecessarily inefficient -- I think we need to figure out how to directly generate the target indexes on GPU.